dev_guide/builds/build_troubleshooting.adoc

```diff
@@ -0,0 +1,26 @@
+[[dev-guide-build-troubleshooting]]
+= Build Troubleshooting
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+[cols="1,4",options="header"]
+.Troubleshooting Guidance for Builds
+|===
+|Issue |Resolution
+a|A build fails with:
+
+----
+requested access to the resource is denied
+----
+a|You have exceeded one of the xref:../../dev_guide/compute_resources.adoc#dev-guide-compute-resources[image
+quotas] set on your project. Check your current quota and verify the limits
+applied and storage in use:
+
+----
+$ oc describe quota
+----
+|===
```

Comment:
Agreed, though I couldn't remember if someone had already taken it from section -> table, and I'd be putting it back into sections again. But I do think the table is weird, I'll change it.